build.gradle

```diff
@@ -69,8 +69,8 @@ repositories {
 dependencies {
     minecraft "net.minecraftforge:forge:${minecraft_version}-${forge_version}"
 
-    compileOnly fg.deobf("_:rubidium:0.5.6")
-//    implementation fg.deobf("_:rubidium:0.5.6")
+    compileOnly fg.deobf("_:rubidium:mc1.19.2-0.6.2c")
+//    implementation fg.deobf("_:rubidium:mc1.19.2-0.6.2c")
```

Comment:
Latest version of Rubidium available for 1.19.2 at the point I started working on this